test/e2e-upgrade-tests.sh

```diff
@@ -42,6 +42,8 @@ initialize $@ --skip-istio-addon
 
 TIMEOUT=${TIMEOUT:-30m}
 
+run_postinstall_V015 || fail_test 'Running postinstall failed'
```

Comment:
Just curious why this is necessary? We're installing the v15 fresh, so there shouldn't need to be a reason for this?

Reply:
true :-)